client/src/theme/Components/reader/_toc-drawer.scss

```diff
@@ -87,7 +87,7 @@
       > ul > li {
```

Comment:
This works great! However, there's an odd appearance at certain widths (between 1024px and about 600px) where the nested list padding is less than the padding of the list above it. This may have always been the case, but we should wrap it up here. You should be able to fix this in the tocBasePadding mixin. Please have a look and let me know if you have any questions.
![manifold scholarship 2017-04-19 11-52-01](https://cloud.githubusercontent.com/assets/1628239/25196892/e4f1d89c-24f6-11e7-85fc-1afbfcbc5239.png)